### DISCUSSION TITLE
第4节课作业(22班) 

### DISCUSSION THREAD

**Discussion by BarryYin** [InternLM/Tutorial#237] [作业]
在这里提交链接

**Comment by miku8miku** [comment]
https://note.youdao.com/s/5Cc7Op2Y

**Comment by lilxmx** [comment]
https://blog.csdn.net/L1142456160/article/details/135632018

**Comment by liuwei16** [comment]
https://g5b4uupmby.feishu.cn/wiki/QFEKwUNrfi4bXukLAL9cDwEvnOc

**Comment by jiayaozhang** [comment]
https://github.com/jiayaozhang/InternLM_Notes/blob/main/Lecture4.md

**Comment by lankuohsing** [comment]
https://github.com/lankuohsing/InternLM_notes/blob/main/lecture_4/homework_4.md

**Comment by lyyilin** [comment]
https://blog.csdn.net/qq_73668945/article/details/135563747

**Comment by Jasimyn** [comment]
https://broadleaf-gemini-274.notion.site/Lecture4-e88ab53de5da4007aa816e9c8db6f936?pvs=4

**Comment by cavedweller509** [comment]
https://github.com/BestAnHongjun/InternLM_Notes/blob/main/homework/homework_4.md

**Comment by mmb135** [comment]
https://www.yuque.com/r/note/4c4e840f-431f-49ae-8a1f-670802d21385

**Comment by funcfang** [comment]
https://www.yuque.com/funcfang/ke5qbq/rtdgw75kcufqp2l8?singleDoc#

**Comment by CDL0726** [comment]
https://github.com/CDL0726/InternLM_Notes/tree/Assignment

**Comment by linkch0** [comment]
https://linkch0.github.io/courses/InternLM/assignment/#l4-xtuner
